addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -103,6 +103,7 @@ def button_validate(self):
                 new_landed_cost_value = line.move_id.landed_cost_value + line.additional_landed_cost
                 line.move_id.write({
                     'landed_cost_value': new_landed_cost_value,
+                    'value': line.move_id.value + cost_to_add,
```

Comment:
In the migration scripts, I supposed the value was without landed costs, so I subtracted it again.  Need to be clear about this.